light/light_test.go

```diff
@@ -136,3 +142,18 @@ func TestClientIntegration_VerifyLightBlockAtHeight(t *testing.T) {
 
 	require.EqualValues(t, 3, h.Height)
 }
+
+func waitForBlock(ctx context.Context, p provider.Provider, height int64) (*types.LightBlock, error) {
```

Comment:
why not make this a public function or a method on provider? it it's used in the example then people will need to go hunting for it.

Reply:
True but it's used as part of the testing scaffold to ensure that the Tendermint node has reached a sufficient height for the test to deterministically pass. As this isn't really a test and we're not really asserting anything we could remove it and just plainly run the light client functions